src/components/VContentSwitcher/VSearchTypes.vue

```diff
@@ -18,11 +26,41 @@
       :selected="item === activeItem"
       @click="handleClick(item)"
     />
+    <div
+      v-if="content.additionalTypes && content.additionalTypes.length > 0"
+      class="mt-2"
+      :class="{
+        'bg-dark-charcoal-06 border-t border-dark-charcoal-20': !bordered,
+      }"
+    >
+      <h4
+        :class="{
+          'ps-0': !bordered,
+          'ps-6': !bordered,
```

Comment:
These two classes seem to be overriding each other.

Reply:
Fixed, thanks for catching this!